portal/models/intervention_strategies.py

```diff
@@ -463,14 +463,20 @@ def completed_card_html(assessment_status):
                     name=user.display_name,
                     registry=assessment_status.assigning_authority)
             else:
+                greeting = _(u"Hi, {}").format(user.display_name)
```

Comment:
I've recently remembered `_()` accepts format strings and keyword arguments to `_()`. 
This would give the translators a little more context, and would be recognized as a "placeholder" in Smartling, (as opposed to `{}`), preventing them from editing the placeholder:
```
_(u"Hi, %(full_name)s", full_name=user.display_name)
```

Reply:
@ivan-c thank you for the tip, ivan!